provision/aws-instance/aws-forensic-architecture.tf

```diff
@@ -1,60 +1,53 @@
-variable "region" {
```

Comment:
Just a thought- would it make more sense to call this file 'aws-timemap.tf'? And the same with the corresponding .tf file in gcp.